org_status/org_status.py

```diff
@@ -195,6 +214,18 @@ def main():
             all_repositories += org_host.repositories
             continue
 
+        if args.import_repos:
```

Comment:
One of the core reasons we want to have this feature implemented is so we can skip fetching the org hosts for a list of all the repos under an org each time. Please make sure that the for loop above actually doiing this doesn't run if `args.import_repos` is True.

One other smart way to approach this is to fix the org host class to support building it using raw properties (i.e. directly pass repositories list to it so it doesn't have to query the service), then fix `org_host = Host(token, org, verbose=args.verbose)`.